sagemaker-featurestore/feature_store_object_detection_ground_truth.ipynb

```diff
@@ -0,0 +1,484 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Cast columns of df of type object to string. \n",
+    "df = cast_object_to_string(df)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create Feature Group and Ingest data into it\n",
+    "\n",
+    "Below we start by appending the `EventTime` feature to your data to timestamp entries, then we load the feature definition, and instantiate the Feature Group object. Then lastly we ingest the data into your feature store."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from time import gmtime, strftime, sleep\n",
+    "\n",
+    "feature_group_name = 'ground-truth-classification-feature-group-' + strftime('%d-%H-%M-%S', gmtime())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Instantiate a `FeatureGroup` object for your data. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker.feature_store.feature_group import FeatureGroup\n",
+    "\n",
+    "feature_group = FeatureGroup(name=feature_group_name, sagemaker_session=sagemaker_session)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "record_identifier_feature_name = \"item_name\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Append the `EventTime` feature to your data frame. This parameter is required, and time stamps each data point."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "\n",
+    "current_time_sec = int(round(time.time()))\n",
+    "\n",
+    "event_time_feature_name = \"EventTime\"\n",
+    "# append EventTime feature\n",
+    "df[event_time_feature_name] = pd.Series([current_time_sec]*len(df), dtype=\"float64\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Load Feature Definition's of your data into your feature group."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feature_group.load_feature_definitions(data_frame=df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create your feature group.\n",
+    "\n",
+    "**Important**: You will need to substitute your KMS Key ARN for `kms_key` for server side encryption. "
```

Comment:
"**Important**: You must substitute your KMS key ARN for `kms_key` for server-side encryption. "